src/pw_utils.rs

```diff
@@ -619,31 +653,18 @@ impl PipeWire {
                         let plane_count = dmabuf.num_planes();
                         assert_eq!((*spa_buffer).n_datas as usize, plane_count);
 
-                        for (i, (fd, (stride, offset))) in
-                            zip(dmabuf.handles(), zip(dmabuf.strides(), dmabuf.offsets()))
-                                .enumerate()
-                        {
+                        for (i, fd) in dmabuf.handles().enumerate() {
```

Comment:
I'll revert this (and the related changes) back once I find why it causes issues